Controllers/controller_mppi.py

```diff
@@ -56,42 +70,35 @@
 ekc_noise = 0.0  # 0.2
 cc_noise = 0.0  # 0.2
 
-"""Parameters weighting the different cost components"""
-dd_weight = 5.0e1*(1+dd_noise*np.random.uniform(-1.0, 1.0))
-ep_weight = 5.0e4*(1+ep_noise*np.random.uniform(-1.0, 1.0))
-ekp_weight = 1.0e-2*(1+ekp_noise*np.random.uniform(-1.0, 1.0))
-ekc_weight = 5.0e0*(1+ekc_noise*np.random.uniform(-1.0, 1.0))
-cc_weight = 1.0e0*(1+cc_noise*np.random.uniform(-1.0, 1.0))
```

Comment:
@marcinpaluch1994 Important: If some randomized weights are needed here, we need to re-add them! I just removed them because it wouldn't work with the GUI..?

Reply:
For me for the moment is fine. I added this randomised weights to cover state-space better, not sticking to trajectories for particular controller. But I will be not back to training I guess for the next two weeks at least (now holidays, afterwards I will probably work on documentation) so do what you. If I need it at any time in future I can read it easily.

Reply:
I've changed this once more s.th. it reads the weights from config and if you set some noise factor >0, it'll disturb the weight. That way we have the best of both worlds :)